tests/e2e_tests/client_utils.py

```
@@ -0,0 +1,357 @@
+                        "score": hit.score,
+                        "payload": hit.payload,
+                        "vector": hit.vector
+                    } for hit in results.points
+                ],
+                "status": "ok",
+                "time": 0
+            }
+            
+        except Exception:
+            print("Search failed")
+            raise RuntimeError("Search failed")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)